app/src/main/assets/DOC_HTML/apps/J_stepper.md

```diff
@@ -0,0 +1,6 @@
+Stepper Motors
+---
+
+* Connect the stepper motor with the SQR pins appropriately.
```

Comment:
`Appropriately` means how? Can you please go through the implemented code and try to derive the documentation accordingly?